### ISSUE TITLE
[CODE] Make selected freshkill feeding tactic jump to top of the list in clearing

### ISSUE THREAD

**Issue by j-gynn** [ClanGenOfficial/clangen#2418]
**Description**
When on the freshkill screen, it's kind of annoying to have to scroll down if you've selected one of the lower tactics, and can make me personally think that I broke something in the game. If it could be moved to the top when you select it, I think that could be a good improvement?

Mockup
![selected_tactic_jump_to_top](https://github.com/ClanGenOfficial/clangen/assets/48025294/2a84ab54-2e4c-441d-b03c-af7aa1b0d262)

**Comment by scribblecrumb** [comment]
Good idea!  Personally I'd like to move the tactic stuff in general into a pop-up window instead of trying to fit it into that short box.  It's just a bit of an awkward shape for the info it's trying to provide.

**Comment by j-gynn** [comment]
Agreed. As part of my semi-impulsive freshkill refactor, a fun organic property has emerged in that it would be quite feasible to allow users to create custom priorities by combining the current sorting options atomically. Obviously impossible given the current spatial limitations, but a fun possibility nonetheless!

**Comment by j-gynn** [comment]
Closing as irrelevant following the clearing rework
